scripts/run_validator.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
-tag=latest
+tag=add-nhse-theme
```

Comment:
Will this need to be reverted to latest?

Reply:
Reverted now